dev/com.ibm.ws.jdbc_fat_derby/test-applications/trandriver/src/jdbc/tran/none/driver/TranNoneConnection.java

```diff
@@ -298,7 +298,7 @@ public void setSchema(String arg0) throws SQLException {
 
     @Override
     public void setTransactionIsolation(int arg0) throws SQLException {
-        throw new SQLException("setTransactionIsolation called with " + arg0);
+        impl.setTransactionIsolation(arg0);
```

Comment:
By changing this we are losing some test coverage to ensure we aren't calling this method when we shouldn't be.  Can you update this so we only allow isolation levels to be set that aren't the default.